src/main/webapp/app/entities/participation/participation.component.html

```diff
@@ -29,7 +29,7 @@ <h2>
                 <td>{{participation.initializationState}}</td>
                 <td>{{participation.initializationDate | date:'medium'}}</td>
                 <td>
-                    <a routerLink="/user-management/{{participation.student?.login}}">{{participation.student?.firstName}} {{participation.student?.lastName}}</a>
+                    <a routerLink="/user-management/{{participation.student?.login}}/edit">{{participation.student?.firstName}} {{participation.student?.lastName}}</a>
```

Comment:
The page behind this link should only be visible to admins (the corresponding REST call can also only be executed by admins, see UserResource). It would be better to navigate to /view which can be visible to tutors, instructors and admins